vignettes/Workflow.Rmd

```diff
@@ -0,0 +1,495 @@
+---
+title: "Some Basic Analysis of ChIP-Seq Data"
+converter:
```

Comment:
Replace `converter` with `author`. This is currently preventing your name from appearing on the HTML.